R/tm_g_distribution.R

```diff
@@ -1302,7 +1338,7 @@ srv_distribution <- function(id,
           card$append_text("Comment", "header3")
           card$append_text(comment)
         }
-        card$append_src(teal.code::get_code(output_q()))
+        card$append_src(teal.code::get_code(req(decorated_output_q())))
```

Comment:
`decorated_output_q` is not created anywhere

Reply:
Should be fixed with last commits